Tests/SwiftGRPCTests/ClientTestExample.swift

```diff
@@ -34,6 +34,12 @@ fileprivate class ClientUnderTest {
     return try service.get(Echo_EchoRequest(text: input)).text
   }
   
+  func getWordasynchronous(_ input:String,completion:@escaping (_ word:String)->()) throws  {
```

Comment:
Given that this method is only being used once, how about removing it and directly calling `try service.get` in `testClientAsynchronous`?

Also, please bring your whitespace more in line with the rest of our codebase.

Reply:
Sure, but it will be `try client.service.get`